webapi/src/main/scala/org/knora/webapi/slice/api/v3/resources/ResourcesRestServiceV3.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2021 - 2025 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.api.v3.resources
+import zio.*
+
+import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.slice.admin.domain.model.KnoraProject
+import org.knora.webapi.slice.admin.domain.model.KnoraProject.ProjectIri
+import org.knora.webapi.slice.admin.domain.service.KnoraProjectService
+import org.knora.webapi.slice.api.v3.NotFound
+import org.knora.webapi.slice.api.v3.OntologyAndResourceClasses
+import org.knora.webapi.slice.api.v3.ResourceClassAndCountDto
+import org.knora.webapi.slice.api.v3.V3ErrorInfo
+import org.knora.webapi.slice.api.v3.ontology.OntologyRestServiceV3
+import org.knora.webapi.slice.common.KnoraIris.ResourceClassIri
+import org.knora.webapi.slice.ontology.domain.service.OntologyRepo
+import org.knora.webapi.slice.resources.repo.service.ResourcesRepo
+
+class ResourcesRestServiceV3(
+  projectService: KnoraProjectService,
+  ontologyRepo: OntologyRepo,
+  ontologyRestService: OntologyRestServiceV3,
+  resourcesRepo: ResourcesRepo,
+)(implicit val sf: StringFormatter) {
```

Comment:
As above: I would not expect a RestService to call into both other RestServices and Repos